test/e2e/apimachinery/garbage_collector.go

```diff
@@ -330,7 +330,14 @@ func newCronJob(name, schedule string) *batchv1beta1.CronJob {
 
 var _ = SIGDescribe("Garbage collector", func() {
 	f := framework.NewDefaultFramework("gc")
-	It("should delete pods created by rc when not orphaning", func() {
+
+	/*
+		    Testname: garbage-collector-delete-rc-orphaning-false
+		    Description: Ensure that if deleteOptions.OrphanDependents is set to false,
```

Comment:
OrphanDependents is deprecated. We should update the test to use deleteOptions.PropagationPolicy (https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L455). Could you update the test to use the getBackgroundOptions() instead of getNonOrphanOptions()? That's just an API change, shouldn't cause any behavior change.